gateway-ha/pom.xml

```diff
@@ -19,6 +19,25 @@
         <app.main.class>io.trino.gateway.ha.HaGatewayLauncher</app.main.class>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-dependencies</artifactId>
+                <version>${dropwizard.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>${dep.junit.version}</version>
+                <type>pom</type>
+                <scope>test</scope>
```

Comment:
I am not sure scope is correct in depMgt .. I think the scope needs to be declared in dependencies

Reply:
Yep. You shouldn't define the scope in the <dependencyManagement> section. These are just version definitions that you need to apply to a given scope anyway.